### PR TITLE
Add a table for Remote Playback controls and states

### PR DESCRIPTION
**Pull request by takumif**
This addresses #148. AFAICT none of the fields is required to be in the initial state sent, but please let me know if I misunderstood this.

***
<a href="https://pr-preview.s3.amazonaws.com/takumif/openscreenprotocol/pull/217.html" title="Last updated on Sep 15, 2019, 9:49 PM UTC (d0d213b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webscreens/openscreenprotocol/217/dea1dce...takumif:d0d213b.html" title="Last updated on Sep 15, 2019, 9:49 PM UTC (d0d213b)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1234,7 +1231,25 @@ intentionally mirror settable attributes and methods of the
...
+<tr><td>loop</td>                   <td>False</td>                                      <td>Required</td></tr>
+<tr><td>paused</td>                 <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>muted</td>                  <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>volume</td>                 <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>seek</td>                   <td>-</td>                                          <td>Required</td></tr>
+<tr><td>fast-seek</td>              <td>-</td>                                          <td>Required</td></tr>
+<tr><td>playback-rate</td>          <td>Decided by the receiver</td>                    <td>Not required</td></tr>
+<tr><td>poster</td>                 <td>Decided by the receiver</td>                    <td>Not required</td></tr>
+<tr><td>enabled-audio-track-ids</td><td>-</td>                                          <td>Required</td></tr>
+<tr><td>select-video-track-id</td>  <td>-</td>                                          <td>Required</td></tr>
```
Comment (pthatcherg) [inline index.bs]: This should be selected-video-track-id, as it is in the .cddl file.  Could you change it (and the other place in this file, which is a mistake)?

File: index.bs [hunk 2]
```diff
@@ -1234,7 +1231,25 @@ intentionally mirror settable attributes and methods of the
...
+<tr><td>added-text-tracks</td>      <td>-</td>                                          <td>Not required</td></tr>
+<tr><td>changed-text-tracks</td>    <td>-</td>                                          <td>Not required</td></tr>
```
Comment (pthatcherg) [inline index.bs]: Perhaps "none" or "empty" instead of "-" would be more clear.

File: index.bs [hunk 3]
```diff
@@ -1234,7 +1231,25 @@ intentionally mirror settable attributes and methods of the
...
+<tr><td>preload</td>                <td>Decided by the receiver</td>                    <td>Not required</td></tr>
+<tr><td>loop</td>                   <td>False</td>                                      <td>Required</td></tr>
+<tr><td>paused</td>                 <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>muted</td>                  <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>volume</td>                 <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>seek</td>                   <td>-</td>                                          <td>Required</td></tr>
+<tr><td>fast-seek</td>              <td>-</td>                                          <td>Required</td></tr>
+<tr><td>playback-rate</td>          <td>Decided by the receiver</td>                    <td>Not required</td></tr>
+<tr><td>poster</td>                 <td>Decided by the receiver</td>                    <td>Not required</td></tr>
+<tr><td>enabled-audio-track-ids</td><td>-</td>                                          <td>Required</td></tr>
```
Comment (pthatcherg) [inline index.bs]: These IDs are for the *controls*, not the *state*.  There are 2 different messages.   Maybe we should have a table for both, but the issue/TODO was for the state table, so we need at least that.

File: index.bs [hunk 4]
```diff
@@ -1234,7 +1231,25 @@ intentionally mirror settable attributes and methods of the
     if a cue ID is invalid (removing an un-added ID or adding an ID twice, for example),
     the receiver may reject the text track change.
 
-Issue: Add a table for whether it's required and what the default is.
+<div class="note">
+<table>
+<tr><th>Field</th>                  <th>Default value</th>                              <th>Receiver support</th></tr>
```
Comment (pthatcherg) [inline index.bs]: Should make it clear this is the default for the *initial* controls.  For all other controls, the default is "unchanged".

Comment (takumif) [reply index.bs]: Updated.

**Comment by pthatcherg** [comment]
I think this is good enough to merge.  We can tweak it further if necessary later.

**Pull request merged**